tensorboard/plugins/audio/audio_demo.py

```diff
@@ -31,193 +31,193 @@
                        'Directory into which to write TensorBoard data.')
 
 tf.flags.DEFINE_integer('steps', 500,
-                        'Number of frequencies of each waveform to generate.')
+                                                'Number of frequencies of each waveform to generate.')
```

Comment:
What? Why?

It looks like the formatter didn't realize that this is hanging alignment and not start-of-statement indentation?